app/src/main/res/layout/walkthrough_topic_header_view.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <data>
+
+    <variable
```

Comment:
Remove `data` as `ViewModel` is not getting used

Reply:
how to set then registerViewDataBinder couldn't figure out that part which why I kept this in the first place

Reply:
@aggarwalpulkit596 Yeah, actually this is useless at the same time we cannot remove it as it i needed for data binding. Do not remove this. Instead in future will be optimise on `BindableDataAdapter`. Thanks for clarifying.